app/assets/stylesheets/application.scss

```diff
@@ -13,5 +13,5 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
-
+ *= require font_awesome_import
```

Comment:
based on the documentation, it looks like this should read as:
```
*= require font-awesome
```
I recently made a staff announcement that these requires should be in a different order (not your fault, it's the way the boilerplate was generated)
```
 *= require font-awesome
 *= require foundation_and_overrides
 *= require_tree .
 *= require_self
```
take a look at my staff announcement post for why we're making this change!

If you're not using any font awesome icons just yet, you don't necessarily need to add this gem. One reason for this is that we want to make sure we're importing and using the library correctly to see the icons!